README.md

```diff
@@ -168,15 +168,24 @@ HEAP_DUMP_PATH 	| Specify a location for a heap dump using `XX:HeapDumpPath` | `
+CASSANDRA_KEYSPACES_PREFIX | Specify a prefix to use when creating Pega-managed keyspaces in Cassandra.                                                                                                                                                                                                                                                                                                                |
+CASSANDRA_EXTENDED_TOKEN_AWARE_POLICY | Enable an extended token aware policy for use when a Cassandra range query runs. When enabled this policy selects a token from the token range to determine when Cassandra node to send the request. Before you can enable this policy, you must configure the token range partitioner.                                                                                                   | `false`
+CASSANDRA_LATENCY_AWARE_POLICY | Enable a latency awareness policy, which collects the latencies of the queries for each Cassandra node and maintains a per-node latency score (an average).                                                                                                                                                                                                                               | `false`
+CASSANDRA_CUSTOM_RETRY_POLICY | Enable the use of a customized retry policy for your Pega Platform deployment. After enabling this policy number of retires in Pega Platform should be configured using the dynamic system setting (DSS): `dnode/cassandra_custom_retry_policy/retryCount`, default value is `1`. When enabled, your deployment will retry using the custom value you set in the DSS.                         | `false`
+CASSANDRA_SPECULATIVE_EXECUTION_POLICY | Enable the speculative execution policy for retrieving data from your Cassandra service. When enabled, Pega Platform can send a query to multiple nodes in your Cassandra service and process the first query response. This provides lower perceived latencies at the client, but puts greater load on your Cassandra service.                                                           | `false`
+CASSANDRA_JMX_METRICS_ENABLED | Enable reporting of DDS SDK metrics to a Java Management Extension (JMX) format for use by your organization to manage your Cassandra service. Setting this property `false` will disable metrics being exposed via the JMX interface, and also limit metrics being collected via the Decision Data Store (DDS) landing page.                                                             | `true`
+CASSANDRA_CSV_METRICS_ENABLED | Enable reporting of DDS SDK metrics to a Comma Separated Value (CSV) format for use by your organization to monitor your Cassandra service. The location that CSV files will be written to can be specified using the dynamic system setting (DSS): dnode/ddsclient/metrics/csv_directory. If not configured, the CSV files will be written to the Pega Platform specific work directory. | `false`
```

Comment:
Enable reporting of DDS SDK metrics to a Comma Separated Value (CSV) format for use by your organization to monitor your Cassandra service. If you enable this property, use the Pega Platform DSS: dnode/ddsclient/metrics/csv_directory to customize the filepath to which the deployment sends CSV files. By default, after you enable this property, this DSS is set to write the CSV files to the Pega Platform specific work directory.